src/librustc_mir/const_eval.rs

```diff
@@ -1,30 +1,31 @@
 // Not in interpret to make sure we do not use private implementation details
 
-use std::fmt;
-use std::error::Error;
 use std::borrow::{Borrow, Cow};
-use std::hash::Hash;
 use std::collections::hash_map::Entry;
 use std::convert::TryInto;
+use std::error::Error;
+use std::fmt;
+use std::hash::Hash;
 
+use crate::interpret::eval_nullary_intrinsic;
 use rustc::hir::def::DefKind;
 use rustc::hir::def_id::DefId;
-use rustc::mir::interpret::{ConstEvalErr, ErrorHandled, ScalarMaybeUndef};
 use rustc::mir;
-use rustc::ty::{self, Ty, TyCtxt, subst::Subst};
-use rustc::ty::layout::{self, HasTyCtxt, LayoutOf, VariantIdx};
+use rustc::mir::interpret::{ConstEvalErr, ErrorHandled, ScalarMaybeUndef};
 use rustc::traits::Reveal;
+use rustc::ty::layout::{self, HasTyCtxt, LayoutOf, VariantIdx};
+use rustc::ty::{self, subst::Subst, Ty, TyCtxt};
 use rustc_data_structures::fx::FxHashMap;
-use crate::interpret::eval_nullary_intrinsic;
 
-use syntax::{source_map::{Span, DUMMY_SP}, symbol::Symbol};
+use syntax::{
+    source_map::{Span, DUMMY_SP},
+    symbol::Symbol,
+};
 
-use crate::interpret::{self,
-    PlaceTy, MPlaceTy, OpTy, ImmTy, Immediate, Scalar, Pointer,
-    RawConst, ConstValue, Machine,
-    InterpResult, InterpErrorInfo, GlobalId, InterpCx, StackPopCleanup, AssertMessage,
-    Allocation, AllocId, MemoryKind, Memory,
-    snapshot, RefTracking, intern_const_alloc_recursive,
+use crate::interpret::{
+    self, intern_const_alloc_recursive, snapshot, AllocId, Allocation, AssertMessage, ConstValue,
+    GlobalId, ImmTy, Immediate, InterpCx, InterpErrorInfo, InterpResult, MPlaceTy, Machine, Memory,
+    MemoryKind, OpTy, PlaceTy, Pointer, RawConst, RefTracking, Scalar, StackPopCleanup,
 };
```

Comment:
This will conflict real bad with https://github.com/rust-lang/rust/pull/67327... maybe land that first?

Reply:
Yea I guess... I just got so annoyed at autoformatting operating on these files...